exporter/clickhousemetricsexporter/clickhouse.go

```diff
@@ -235,7 +241,7 @@ func (ch *clickHouse) Write(ctx context.Context, data *prompb.WriteRequest, metr
 			fingerprintToName[f] = make(map[string]string)
 		}
 		fingerprintToName[f][nameLabel] = metricName
-		fingerprintToName[f][env] = env
```

Comment:
Was this a bug earlier as I see map key is same as value `env` ? If not, what changed here?

Reply:
Yes, this was a bug